src/Tribe/Main.php

```diff
@@ -286,6 +286,23 @@ public function load_assets() {
 			]
 		);
 
+		tribe_asset(
+			$this,
+			'tribe-admin-image-field',
+			'admin-image-field.js',
+			[ 'jquery' ],
+			null,
+			[
+				'localize' => [
+					'name' => 'tribe_admin_image_field',
+					'data' => [
+						'select_image_text' => _x( 'Select an image to use in your email headers', 'Select image text.', 'tribe-common' ),
+						'use_image_text'    => _x( 'Use this image', 'Select image text.', 'tribe-common' ),
+					]
+				]
```

Comment:
Move the actual translations to the field itself as `data-` params. Allows us to have diff translations to each field.